_employment/job-seekers/unemployment-support.md

```diff
@@ -34,7 +34,7 @@ You will need the following documents when you apply:
 - Social security card 
 - Current résumé or military job history 
 
-If you are unemployed and at risk of becoming homeless, [VA can help](vets.gov homeless section). 
+If you are unemployed and at risk of becoming homeless, [VA can help]({{site.baseurl}}/dummy-placeholder.html). 
```

Comment:
Do you need the {{site.baseurl}} everywhere?  These are just server-relative links so I think you can do `/_dummy-placeholder.html`.